apps/system/js/remote/system_remote.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+(function(exports) {
+  var SystemRemote = {
+    DEBUG: false,
+
+    REQUIRED_MODULES: ['SimpleAppWindowManager', 'WallpaperManager'],
+    REGISTERED_EVENTS: ['wallpaperchange'],
+
+    _started: false,
+    _readyCallback: null,
+
+    debug: function(msg) {
+      if (this.DEBUG && msg) {
+        this.printLog.apply(this, arguments);
+      }
+    },
+
+    printLog: function() {
+      var arg = ['[SystemRemote #' + this.displayId + '] ' + arguments[0]]
+        .concat(Array.from(arguments).slice(1));
+      console.log.apply(console, arg);
+    },
+
+    ready: function(callback) {
```

Comment:
I forgot to remove it. I intended to use this patten in other modules but it has been dropped then.